README.md

```diff
@@ -1,11 +1,12 @@
 # CircleCI Concurrency Control Orb
 
+[![CircleCI](https://circleci.com/gh/eddiewebb/circleci-queue/tree/master.svg?style=svg)](https://circleci.com/gh/eddiewebb/circleci-queue/tree/master) [![GitHub license](https://img.shields.io/github/license/eddiewebb/circleci-queue)](https://github.com/eddiewebb/circleci-queue/blob/master/LICENSE)
```

Comment:
Change circleci status badge to use sheilds.io as well for consistent style

Reply:
Changed circleci status badge